packages/theme/src/types.ts

```diff
@@ -16,7 +16,7 @@
 
 import { Theme } from '@material-ui/core';
 
-export type BackstageTheme = Theme & {
+export type BackstageThemeType = Theme & {
```

Comment:
Should stick with BackstageTheme imo, suffixing types with Type is a no-no according to MS iirc. instances should be changed to use camelCase though :D

Reply:
Agreed. That part was a bit of a mess in terms of the types tbh... I don't know what the best approach is